client/app/scripts/liveblog-edit/views/filter-by-member.html

```diff
@@ -0,0 +1,41 @@
+<button ng-click="vm.toggleSelector()" class="btn btn-info btn-mini btn--plus"><i class="icon-white icon-plus-large"></i></button>
+<div class="dropdown-content" ng-click="$event.stopPropagation();" ng-if="vm.openSelector">
+    <div class="user-select">
+        <div>
+            <form>
+                <label for="assignee-search" translate>Select users</label>
+                <input type="text" name="assignee-search" ng-model="vm.search">
+            </form>
+            <div class="search-result">
+                <ul class="users-list-embed">
+                    <li ng-repeat="user in vm.members | filter:vm.search | orderBy: 'display_name'"
```

Comment:
It is strongly recommended to use `track by` for collections that tend to change to avoid unnecessary DOM operations, and search results are a good candidate for that in most cases.